.github/workflows/lint.yml

```diff
@@ -0,0 +1,49 @@
+name:                              Check style
+
+on:
+# Uncomment when needed
```

Comment:
So we wouldn't run the RustFmt and Clippy stages unless this is uncommented, right?

Reply:
Yes, only the night build, but it will be possible to configure it further

Reply:
Can we split `rust-fmt` and `clippy` and run `fmt` on every push & pr?

`clippy` is not part of our workflow yet, but `fmt` is crucial to be checked on every PR.